website/content/docs/kubernetes/resource-status.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Kubernetes - Resource Status
+description: |-
+  Waypoint tracks the Kubernetes resources created during deployment and can show you their details and health status in the web UI and CLI.
+---
+
+# Resource Status
+
+Waypoint tracks the Kubernetes resources created during deployment
+and can show you their details and health status in the web UI and CLI.
+This helps show that what Waypoint is doing isn't magic and isn't trying
+to hide anything, and lets developers and operators more easily debug
+their deployments when something isn't healthy.
+
+<!-- TODO: screenshot -->
+
+## Viewing Resource Status
+
+TODO: UI
+
+You may also view the resource status in the CLI. While in your project
+repository (with a `waypoint.hcl` file), execute `waypoint status`. To
+view detailed status about a specific app, specify the `-app` flag with
+the name of the application.
+
+```shell-session
+$ waypoint status
+TODO
+```
+
+## Refreshing Resource Status
+
+When viewing the resource status in the UI or the CLI, Waypoint by default
+will show you the _most recent status report_, but this report may be minutes
+old. You can request a refreshed status report in the CLI by specifying the
+`-refresh` flag. Note that this may cause the `status` operation to take
+anywhere from a few seconds to a few minutes depending on the size of your
+deployment and the load on Kubernetes.
+
+```shell-session
+$ waypoint status -refresh
+TODO
+```
+
+-> **Refresh functionality is not available in the UI, yet.** Unfortunately,
+you can't currently force a status report refresh in the web UI. We will
+implement this soon.
```

Comment:
You can refresh statuses in the UI! It only works for GitOps backed projects though at the moment.